pkg/utils/util.go

```diff
@@ -18,9 +19,25 @@ import (
 )
 
 const (
-	ClustersPageSize        = 50
+	ClustersPageSize             = 50
 	BackplaneApiUrlRegexp string = `(?mi)^https:\/\/api-backplane\.apps\.(.*)`
-	ClusterIDRegexp  string = "/?backplane/cluster/([a-zA-Z0-9]+)/?"
+	ClusterIDRegexp       string = "/?backplane/cluster/([a-zA-Z0-9]+)/?"
+	defaultKubeConfig            = `
+apiVersion: v1
+clusters:
+- cluster:
+    server: https://api-backplane.apps.something.com/backplane/cluster/configcluster
+  name: dummy_cluster
+contexts:
+- context:
+    cluster: dummy_cluster
+    namespace: default
+    user: example.openshift
+  name: default/test123/anonymous
+current-context: default/test123/anonymous
+kind: Config
+preferences: {}
+`
```

Comment:
What is we have an object instead of a hard-coded string here? Would it still function the same?

Reply:
api.config object may do the same but need to find how to convert the object to yaml/json

Reply:
What if `createTempKubeconfig` is modified to function based on a `api.Config` object instead of a string?